client/Dockerfile

```diff
@@ -24,6 +24,8 @@ RUN echo 'server {\
     }\
   }' > /etc/nginx/sites-available/default
 
+RUN mv ./nginx/wasm.conf /etc/nginx/conf.d/wasm.conf
```

Comment:
I would suggest to add a comment here explaining what this line is doing, eg explaining it uses `client/nginx/wasm.conf` for x,y,z

Reply:
Sounds good!